docker/generic/Dockerfile.base

```diff
@@ -0,0 +1,95 @@
+#
+# Install ROS packages used by Autoware.
+#
+ARG ROS_DISTRO
+FROM ros:$ROS_DISTRO
+
+#
+# Install tools and libraries required by Autoware
+#
+RUN apt-get update && apt-get install -y \
+        cmake-curses-gui \
+        cmake-qt-gui \
+        dbus-x11 \
+        dmz-cursor-theme \
+        fonts-dejavu \
+        gconf2 \
+        gnome-terminal \
+        gstreamer0.10-plugins-good \
```

Comment:
This package isn't available in 18.04. It also isn't necessary as a dependency. The only package in Autoware that references it is the pointgrey driver and that driver depends on gscam, which depends on the appropriate version of gstreamer for the selected ROS release/Ubuntu version.

Reply:
@JWhitleyAStuff you're right. The Dockerfile files will also need to be updated. We would need to address #2098 and #2082